ftl/php/layer_builder.py

```diff
@@ -77,9 +78,28 @@ def _gen_composer_install_tar(self, destination_path):
         subprocess.check_call(['rm', '-rf', os.path.join(app_dir, 'vendor')])
 
         with ftl_util.Timing('Composer_install'):
-            subprocess.check_call(
-                ['composer', 'install', '--no-dev', '--no-scripts'],
-                cwd=app_dir)
+            composer_cmd_args = ['composer', 'install', '--no-dev',
+                                 '--no-scripts']
+            proc_pipe = subprocess.Popen(
+                composer_cmd_args,
+                stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                cwd=app_dir
+            )
+            stdout, stderr = proc_pipe.communicate()
+            logging.info("`composer require` stdout:\n%s" % stdout)
+            if stderr:
+                err_txt = "`composer require` had error output:\n%s" % stderr
+                logging.error(err_txt)
```

Comment:
could you embed the `logging.error` call in the constructor of the `UserError`, or wherever the exception is handled? it seems a little weird to both log and raise an exception